pkg/asset/installconfig/aws/metadata.go

```diff
@@ -0,0 +1,35 @@
+package aws
+
+import (
+	"context"
+	"sync"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/pkg/errors"
+)
+
+// Metadata holds additional metadata for InstallConfig resources that
+// does not need to be user-supplied (e.g. because it can be retrieved
+// from external APIs).
+type Metadata struct {
+	session *session.Session
+	sync.Mutex
```

Comment:
since Mutex is `capital`, it can be serialized... which we don't want.

Reply:
> since Mutex is capital, it can be serialized...

Changed to `mutex sync.Mutex` with d399c01ff -> 1b4c1bfa2.